packages/mui-material/src/Modal/Modal.d.ts

```diff
@@ -78,6 +97,98 @@ export interface ModalTypeMap<
         ModalOwnerState
       >;
     };
+    /**
+     * An HTML element or function that returns one.
+     * The `container` will have the portal children appended to it.
+     *
+     * By default, it uses the body of the top-level document object,
+     * so it's simply `document.body` most of the time.
+     */
+    container?: PortalProps['container'];
+    /**
+     * If `true`, the modal will not automatically shift focus to itself when it opens, and
+     * replace it to the last focused element when it closes.
+     * This also works correctly with any modal children that have the `disableAutoFocus` prop.
+     *
+     * Generally this should never be set to `true` as it makes the modal less
+     * accessible to assistive technologies, like screen readers.
+     * @default false
+     */
+    disableAutoFocus?: boolean;
```

Comment:
Are these types duplicated? If I change the description here, does it fail the CI?

If no, why not import from ModalOwnProps?